addons/website_sale/static/src/snippets/s_dynamic_snippet_products/000.js

```diff
@@ -5,14 +5,18 @@ const config = require('web.config');
 const core = require('web.core');
 const publicWidget = require('web.public.widget');
 const DynamicSnippetCarousel = require('website.s_dynamic_snippet_carousel');
+const DynamicSnippet = require('website.s_dynamic_snippet');
 var wSaleUtils = require('website_sale.utils');
 
 const DynamicSnippetProducts = DynamicSnippetCarousel.extend({
     selector: '.s_dynamic_snippet_products',
-    read_events: {
+});
+
+DynamicSnippet.include({
+    read_events: Object.assign({}, DynamicSnippet.prototype.read_events, {
         'click .js_add_cart': '_onAddToCart',
         'click .js_remove': '_onRemoveFromRecentlyViewed',
-    },
+    }),
```

Comment:
Seems strange, especially in stable. Not all dynamic snippets were meant to work with products, right? Should not it be at least only targeting a common Class of all products-related dynamic snippets?

Reply:
I agree this solution is poorly thought. overriding methods in the generic snippet is not a good idea. Would it be okay to move the commit to another branch so that the other fixes can go through? I can fix the current behaviour maybe in the same task where I improve the snippet for new designs. Since it targets a debug feature I think it's not urgent to fix.

Reply:
As you wish but it won't change much to wait to merge all the fixes at the same time either. But as you wish, same for me :+1: